app/utils/cborHelper.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/prefer-default-export */
+import { Buffer } from 'buffer/';
+import { encode, decode } from 'cbor';
+
+/**
+ * Given a string or number, return a buffer containing the value under cbor encoding.
+ * N.B. given a string, this will attempt to convert it to a number, to decrease the encoded size.
+ */
+export function encodeAsCBOR(value: string | number): Buffer {
+    // Prefer saving as numbers:
+    const asNumber = Number(value);
+    if (Number.isInteger(asNumber)) {
+        if (
+            asNumber >= Number.MAX_SAFE_INTEGER ||
```

Comment:
`Number.MAX_SAFE_INTEGER` is also safe, so the equality should be `>` and `<` respectively.